src/Server/TCPServerConnectionFactory.h

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include <Poco/SharedPtr.h>
+#include "Server/TCPProtocolStackData.h"
```

Comment:
there is no need for include. It's only referenced here

Reply:
it is referenced at 27